src/crimson/os/seastore/onode_manager/staged-fltree/node.h

```diff
@@ -152,7 +152,7 @@ class tree_cursor_t final
   eagain_ifuture<> trim_value(context_t, value_size_t);
 
   static Ref<tree_cursor_t> get_invalid() {
-    static Ref<tree_cursor_t> INVALID = new tree_cursor_t();
+    Ref<tree_cursor_t> INVALID = new tree_cursor_t();
```

Comment:
this static cause heap-used-after-free bug, since every get_invalid() will return the same address, and it is freed in Cursor constructor, if next get_invalid() called after that free, it will return a null pointer.  I meet some times but not often before, this times easily trigger that bug, so fix it.

Reply:
Worth to create a dedicated commit for an independent fix